docs/_drafts/scv2-hodometer.md

```diff
@@ -0,0 +1,199 @@
+---
+title: "Seldon Core v2 Deep-Dive---Hodometer"
+tags: seldon seldon-core-v2
+---
+
+Hodometer is an optional component of Seldon Core v2 responsible for collecting anonymous usage metrics.
+The name comes from the [ancient device](https://en.wikipedia.org/wiki/Odometer) for measuring distance, familiar in cars as an _odometer_, because it's all about keeping track of your mileage!
+
```

Comment:
:wrench: Add section on who might be interested in this article, e.g. those looking to understand Core v2 in greater depth, those looking to implement usage metrics in a responsible manner for their own projects, and those simply looking to understand the design decisions in an (until recently) open-source project.

Reply:
@ukclivecox Once I've resolved this comment, that should also address your own previous comment.  Aside from that, I think this nearing completion now as a draft.  (A draft in the Jekyll sense, i.e. that it isn't "published" on the blog, not that it isn't in a good state.)

Reply:
Fixed in https://github.com/agrski/agrski.github.io/pull/40/commits/1ece90a8c3c9ec37389da49be94c16d5d946adc0